src/test/java/webdrivertasks/waitings/WebElementWaiting.java

```diff
@@ -0,0 +1,35 @@
+package webdrivertasks.waitings;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class WebElementWaiting {
+
+    private static final long TIME_OUT = 20;
+
+    private WebElementWaiting() {
+    }
+
+    public static WebElement waitForVisibilityOfWebElement(WebDriver driver, WebElement webElement) {
+       return new WebDriverWait(driver, TIME_OUT)
+                .until(ExpectedConditions.visibilityOf(webElement));
+    }
+
+    public static WebElement waitForElementIsClickable(WebDriver driver, WebElement webElement) {
+        return new WebDriverWait(driver, TIME_OUT)
+                .until(ExpectedConditions.elementToBeClickable(webElement));
+    }
+
+    public static WebElement waitElementFromDropDown(WebDriver driver, String dropDownLocator, String selectedItem) {
```

Comment:
I'm not sure that we need such complicated logic here. Wouldn't it work, if we just wait for the dropdown to appear -> click it -> wait for the needed element to appear - > profit?
One more thing here, I would reflect in the function name, that it works with xpath